code401challenges/linkedLists/src/main/java/linkedLists/LinkedList.java

```diff
@@ -0,0 +1,83 @@
+package linkedLists;
+
+public class LinkedList {
+    Node head; // head of list
+
+    public boolean isEmpty() {
+        return length() == 0;
+    }
+
+    int length() {
+        int length = 0;
+        Node current = head;
+
+        while(current != null){
+            length ++;
+
+            current = current.next;
+        }
+        return length;
+    }
+
+   // Method to insert a new node
+    public LinkedList Insert(LinkedList list, String data)

```

Comment:
In some placed the opening `{` is on the same line as the thing its opening, on others they're placed on the next line. Generally Java syntax dictates that opening `{` sit on the same line.